### PR TITLE
Barrier

### PR DESCRIPTION
**Pull request by oong819**
Add compact, short information about your PR for easier understanding:

add a new block: barrier
like air but walkable
- Goal of the PR
give content creator a tool
- How does the PR work?
new block
- Does it resolve any reported issue?
no
- Does this relate to a goal in [the roadmap](../doc/direction.md)?
no
- If not a bug fix, why is this PR needed? What usecases does it solve?
give content creator a useful tool, can use as a locked door need block place to unlock
## To do

This PR is Ready for Review.

- [ ] List
- [ ] Things
- [ ] To do
make new texture in future
## How to test
/giveme barrier

### CODE REVIEW

**Comment by SwissalpS** [comment]
your locale changes don't seem to have anything to do with the barrier.
Maybe submit that separately, the locale changes have a big chance of being merged.

File: builtin/game/register.lua [hunk 1]
```diff
@@ -379,6 +379,22 @@ core.register_node(":ignore", {
 		return ""
 	end,
 })
+--Stvk pr here
+minetest.register_node(":barrier", {
+	description = S("Barrier"),
+	inventory_image =ignore.png^air.png",
```
Comment (SwissalpS) [inline builtin/game/register.lua:385]: missing opening quotes in line 385

**Comment by Desour** [comment]
There's no reason for this to be in builtin. Just make a mod.

**Comment by oong819** [comment]
the locale is from my first pr leftover, idk how to remove it

**Comment by oong819** [comment]
> your locale changes don't seem to have anything to do with the barrier. Maybe submit that separately, the locale changes have a big chance of being merged.

its is my failed PR leftover because Minetest use Weblate to translate

File: builtin/game/register.lua (hunk 1 above)
Comment (oong819) [reply builtin/game/register.lua:385]: oh its my browser lag sorry, fixing

**Comment by oong819** [comment]
why its still error? ({(){}}) its not missing }

**Comment by oong819** [comment]
anyone help idk how to fix it, i dont see any missing }

**Comment by rubenwardy** [comment]
this should be a mod, not part of builtin

**Comment by SwissalpS** [comment]
@oong819 line 394 has your syntax error. replace ':' with '='

**Comment by oong819** [comment]
> @oong819 line 394 has your syntax error. replace ':' with '='

wow thanks!
finally i can fix it

**Comment by oong819** [comment]
> @oong819 line 394 has your syntax error. replace ':' with '='

its still crash when i load it, do you see any error?

**Comment by SwissalpS** [comment]
have you looked at [maptools] mod? It has many special nodes, probably even the one you are trying to add.
May I suggest you take a closer look at how [maptools] does it and then create your own mod based on that.
We can then debug and discuss further on your repo instead of here where it's offtopic.

**Pull request closed without merging**